types.ts

```diff
@@ -3,7 +3,7 @@ export type ChartPropsData = {
 };
 
 export type QueryTimeObj = {
-  time: number;
+  time: any;
```

Comment:
Can we please put this back to number and see if it breaks? I think we reset it to any when debugging but it wasn't the cause of the bug.